tests/lib/react16.d.ts

```diff
@@ -107,7 +107,6 @@ declare module "react" {
     type NativeUIEvent = UIEvent;
     type NativeWheelEvent = WheelEvent;
 
-    // tslint:disable-next-line:export-just-namespace

```

Comment:
I don't think you should remove these comments. They are here because the whole file was copied from DefinitelyTyped.

Reply:
Reverted